### ISSUE TITLE
Adding Samourai terminology

### ISSUE THREAD

**Issue by rapidlab309** [sparrowwallet/sparrow#669]
I've found out users might miss some of Sparrow's features due to different terms used by the community.
When spending to PayNym, the window suggests to send collaboratively or send directly. Collaboratively refers to the PayNym I wish to spend to or collaboratively with adifferent user? Directly means what in terms of direct/indirect?

I would suggest adding a label that sending collaboratively is Stowaway, while sending directly is Stonewall or StonewallX2.

**Comment by craigraw** [comment]
Yes, I decided not to use Samourai's terminology in the UI intentionally - for the main reason to make these methods sound commonplace and approachable. Like most UX decisions there are pros and cons here, and space and simplicity in the UI prevents me from doing both side by side. 

In this case, 'collaboratively' vs 'directly' refers to how the transaction is constructed. If there are specific examples of confusion around this (e.g. "I thought this button meant something else") please share.

**Comment by RequestPrivacy** [comment]
I know some people who don't use Samourai because they are to lazy to learn all the Samourai terminology. I mean Stonewallx2? Stowaway? Cahoots? Seems as being a to arbitrary naming for people to wrap their head around.

So I guess my view is that people who know these terms have done their homework and will have no problem to use Sparrow even if other names are used. Those who don't know Samourai learn through Sparrow a more generic naming convention which is the better way imho.

**Comment by rapidlab309** [comment]
No doubt Sparrow was made for the mass and not for Samourai users; I understand the UI decisions on this one and I think it's a good choice.

>Craig. I wasn't sure myself, and also another Samourai user, what the differences are between "collaborative" and "direct". For a Samourai user, and I guess it might be for others too, "direct" would be understandable as my wallet to yours (Stowaway if online and listening, BIP47 if not), while "collaborative" could be my wallet & other one's wallet to yours (StonewallX2 if I choose another user, or Stowaway if I choose you as collaborator). I was able to understand that collaborative is Stowaway only when I wasn't able to change the collaborator, and therefore by sending direct I understood I could make it be Stonewall, StonewallX2, as BIP47 TX. I hope it was clear enough for understanding the differences I faced.

>RequestPrivacy. Well, I guess this is what happens when new products are published and yet to be adopted massively. PayNym is also something that wasn't common to read but getting closer to having it in more wallets soon.  I'm not against generic names, not at all. But there's some misunderstanding that users might face, as I and someone else faced, even though we both are Samourai users for years.
